src/test/java/no/unit/bibs/contents/document/handlers/UpdateDocumentTest.java

```diff
@@ -0,0 +1,82 @@
+package no.unit.bibs.contents.document.handlers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import no.unit.bibs.contents.document.DocumentDao;
+import no.unit.bibs.contents.document.DocumentDto;
+import no.unit.bibs.contents.document.DocumentTestBase;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Path;
+import java.time.Instant;
+
+import static no.unit.bibs.contents.CreateContentsApiHandlerTest.CREATE_CONTENTS_EVENT;
+import static no.unit.nva.commons.json.JsonUtils.dtoObjectMapper;
+import static nva.commons.core.ioutils.IoUtils.stringFromResources;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+class UpdateDocumentTest extends DocumentTestBase {
+
+    private UpdateHandler testHandler;
+
+    @BeforeEach
+    void setUp() throws JsonProcessingException, BadRequestException {
+        init();
+        testHandler = new UpdateHandler(documentService, environment);
+        var dto = dtoObjectMapper.readValue(RESOURCE_PATH, DocumentDto.class);
+        when(mockedClient.update(any()))
+            .thenReturn(dto.toDaoBuilder().modified(Instant.now()).build());
+    }
+
+    @Test
+    void getSuccessStatusCodeReturnsOK() {
```

Comment:
Status for update other than just OK? 204 No Content, Accepted etc...

also here do a processinput first